src/components/Kv/KvMap.vue

```diff
@@ -0,0 +1,310 @@
+<template>
+	<div class="tw-relative tw-block tw-w-full" :style="mapDimensions">
+		<div
+			:id="`kv-map-holder-${this.mapId}`"
+			:ref="refString"
+			class="tw-w-full tw-h-full tw-bg-black"
+			:style="{ position: 'absolute' }"
```

Comment:
This was required to override the gl lib css :(, the leaflet based lib obeyed tailwind...